scripts/k-crossgen.cmd

```diff
@@ -2,10 +2,10 @@
 SETLOCAL
 SET ERRORLEVEL=
 
-SET "DOTNET_RUNTIME_PATH=%~dp0."
+SET "KRE_RUNTIME_PATH=%~dp0."
```

Comment:
It's a great idea which makes future renaming much more easier. However, we want to check in the renaming as soon as possible. This can be done in a separate PR.